test/integrations/destinations/ga4/dataDelivery/business.ts

```diff
@@ -0,0 +1,276 @@
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            JSON: validRequest,
+            endpoint: 'https://www.google-analytics.com/debug/mp/collect',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message: '[GA4 Response Handler] - Request Processed Successfully',
+            response: [
+              {
+                error: '{"validationMessages":[]}',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 1,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
+                statusCode: 200,
+              },
+            ],
+            status: 200,
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'ga4_v1_scenario_2',
+    name: 'ga4',
+    description:
+      '[Proxy v1 API] :: Test for a invalid event name - where the destination responds with 200 with error for invalid event name',
+    successCriteria: 'Should return 200 with error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            headers,
+            params,
+            JSON: invalidEventNameRequest,
+            endpoint: 'https://www.google-analytics.com/debug/mp/collect',
+          },
+          metadataArray,
+        ),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            message:
+              'Validation Server Response Handler:: Validation Error for ga4 of field path :events | NAME_INVALID-Event at index: [0] has invalid name [campaign@details]. Only alphanumeric characters and underscores are allowed.',
+            response: [
+              {
+                error:
+                  'Validation Server Response Handler:: Validation Error for ga4 of field path :events | NAME_INVALID-Event at index: [0] has invalid name [campaign@details]. Only alphanumeric characters and underscores are allowed.',
+                metadata: {
+                  attemptNum: 1,
+                  destinationId: 'dummyDestinationId',
+                  dontBatch: false,
+                  jobId: 1,
+                  secret: {},
+                  sourceId: 'dummySourceId',
+                  userId: 'dummyUserId',
+                  workspaceId: 'dummyWorkspaceId',
+                },
+                statusCode: 400,
+              },
+            ],
+            statTags: {
```

Comment:
Check if the `statTags` can be declared as variables and reused

Reply:
addressed